src/content/en/2021/caching.md

```diff
@@ -17,4 +20,435 @@ featured_stat_3: TODO
+     <td>A stale cached entry must be revalidated before its use, via a conditional request.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">no-store</code></td>
+     <td>Indicates that the response must not be cached.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">private</code></td>
+     <td>The response is intended for a specific user and should not be stored by shared caches such as CDNs.</td>
+    </tr>
+    <tr>
+     <td><code class="no-wrap">immutable</code></td>
+     <td>Indicates that the cached entry will never change during its TTL and that revalidation is not necessary.</td>
+    </tr>
+  </table>
+  <figcaption>TODO: Caption the table above</figcaption>
+</figure>
+
+The `max-age` directive is the most commonly found with 62.2% of Desktop requests include a `Cache-Control` response header with this directive.
+
+{{ figure_markup(
+  image="cache-control-directives.png",
+  caption="Usage of `Cache-Control` directives.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1359015817&format=interactive",
+  sheets_gid="1944529311",
+  sql_file="cache_control_directives.sql"
+) }}
+
+Compared to 2020, `max-age` adoption increased by 2% on Desktop and most of the top seven directives in the above chart.
+
+The `immutable` directive is relatively new and can significantly improve cacheability for certain types of requests. However, it is still only supported by a few browsers, and we see most requests from Host Networks like *Wix* with 16.4%, *Facebook* with 8.6%, *Tawk* at 2.8%, and *Shopify* at 2.4%.
+
+The most misused `Cache-Control` directive continues to be `set-cookie`, used for 0.07% of total *directives* for Desktop and 0.08% for Mobile. Although we are seeing a meaningful 0.16% reduction of usage from 2020.
+
+When we take a look when `no-cache`, `max-age=0` and `no-store` are used together, we also see a growing trend year after year, which where `no-store` is specified with either/both of `no-cache` and `max-age=0`, the `no-store` directive takes precedence, and the other directives are ignored. Therefore, driving more awareness on larger conferences using these *directives* can help avoid accidentally wasted bytes.
+
+Fun fact the most common `max-age` value is 30 days, and the largest value is 51 trillion years.
+
+{{ figure_markup(
+  caption="TODO",
+  content="51 trillion years",
+  classes="medium-number",
+  sheets_gid="529870849",
+  sql_file="todo.sql"
+) }}
+
+### `304` Not Modified status
+
+When it comes to size, `304 Not Modified` responses are usually much smaller than `200 OK` responses, so it follows that page performance can be sped up/improved?? by only delivering the necessary size of data. This is where correctly using conditional requests comes in because revalidation (and data savings?) can only be done by using either an `Etag` or `Last-Modified` header.
```

Comment:
@Zizzamia Content of this paragraph needs to be confirmed

When it comes to size, `304 Not Modified` responses are usually much smaller than `200 OK` responses, so it follows that page performance can be sped up/improved?? by only delivering the necessary size of data. This is where correctly using conditional requests comes in because revalidation (and data savings?) can only be done by using either an `Etag` or `Last-Modified` header.